force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +340,15 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
+        Integer wordIndex = -1;
 
         // Loop through the list of words
-            // If the current word is the word to find, exit the loop
+        for (Integer counter=0; counter<words.size(); counter++) {
+            if(words.get(counter) == wordToFind)
+                wordIndex = counter;
+        }
```

Comment:
Try using a for each loop to solve this problem so you don't have to do a words.get() call.